src/lib/modernExtend.ts

```diff
@@ -754,7 +754,11 @@ export function light(args?: LightArgs): ModernExtend {
     const exposes: Expose[] = lightExpose;
 
     if (args.effect) {
-        exposes.push(e.effect());
+        const effects = e.effect();
+        if (args.color) {
+            effects.values.push('colorloop');
```

Comment:
```suggestion
            effects.values.push('start_colorloop', 'stop_colorloop`);
```

`stop_effect` should definitely be here, but since this only stops the color loop, I propose it to rename to `stop_colorloop`

Reply:
Thank you for your review.
Not sure if I got you right: I thought it would be best to extend the existing `stop_effect` (which is added in [`e.effect()`](https://github.com/slugzero/zigbee-herdsman-converters/blob/677035f39a350b2c1f3f7c18ca5c072ccde7df0d/src/lib/exposes.ts#L678)) to also stop the colorloop. 
So you would suggest to rather add a separate `stop_colorloop` which only stops the color loop?

Reply:
You are right, the additional command is also called for lights that don't support colors and fails. I could check with `supportsInputCluster` if the color control cluster is present, but not sure how this behaves e.g. for groups or if there is still a regression for individual devices. 
So I think it is safest to have a separate `stop_colorloop` command as you suggested. Note that I did not change `colorloop` to `start_colorloop` in order to be consistent with Philips devices (where it is called `colorloop` only), and also to be consistent with other Zigbee Gateways (Deconz, zha). If you prefer `start_colorloop`, lmk